neuro_san/service/async_agent_service.py

```diff
@@ -77,11 +77,13 @@ def __init__(self,
 
         self.llm_factory: ContextTypeLlmFactory = MasterLlmFactory.create_llm_factory()
         self.toolbox_factory: ContextTypeToolboxFactory = MasterToolboxFactory.create_toolbox_factory()
-        # Load once and include "agent_llm_info_file" from agent network hocon to llm factory..
+        # Load once and include "agent_llm_info_file" and "agent_toolbox_info_file" from agent network hocon
+        # to llm factory and toolbox factory, respectively.
         agent_network: AgentNetwork = self.agent_network_provider.get_agent_network()
-        agent_llm_info_file = agent_network.get_agent_llm_info_file()
+        agent_llm_info_file: str = agent_network.get_agent_llm_info_file()
+        agent_toolbox_info_file: str = agent_network.get_agent_toolbox_info_file()
```

Comment:
Given the principle of not changing the main interface (see ContextTypeToolboxFactory comments below),
This file would not have to change at all if the information here were coming into the ToolboxFactory contstructor because the load() signature would stay intact.